lib/query.js

```diff
@@ -2,14 +2,18 @@
  * Functions to query properties of nodes
  */
 
+const {print} = require('math-parser')
+const build = require('./build')
+
 export const isIdentifier = node => node && node.type === 'Identifier'
 export const isApply = node => node && node.type === 'Apply'
+export const isParens = node => node && node.type === 'Parentheses'
 
 export const isOperation = node => isApply(node) && !isNumber(node)
 export const isFunction = node => isApply(node) && isIdentifier(node.op)
 
 // TODO: curry it?
-const _isOp = (op, node) => isApply(node) && node.op === op
+export const _isOp = (op, node) => isApply(node) && node.op === op
```

Comment:
Usually names prefixed with `_` are intended to be private.  If we want to make it publish we should change the name.  The current `isOperation` is functionally the same as `isApply` so I think we should make a breaking change and deprecate `isOperation` in its current form and replace `_isOp` with `isOperation`.  We could also call it `isOp` if `isOperation` is too long.